plugins/vpp/plugin_impl_vpp.go

```diff
@@ -590,7 +590,7 @@ func (plugin *Plugin) initSR(ctx context.Context) (err error) {
 	srLogger := plugin.Log.NewLogger("-sr-plugin")
 
 	var stopwatch *measure.Stopwatch
-	if plugin.enableStopwatch {
+	if false {
```

Comment:
Doesn't make much sense.

Reply:
It was a temporary change since this pr solves only acl plugin. But as we talked, I return enableStopwatch so it will be changed back anyway